test/msw-api/setup-worker/notifications.mocks.ts

```diff
@@ -0,0 +1,17 @@
+import { setupWorker, rest } from 'msw'
+
+const worker = setupWorker(
+  rest.get('/user', (req, res, ctx) => {
+    return res(
+      ctx.json({
+        firstName: 'John',
+        age: 32,
+      }),
+    )
+  }),
+)
+
+// @ts-ignore
+window.__MSW_REGISTRATION__ = worker.start({
+  serviceWorker: {},
+})
```

Comment:
I think that i can be 

```suggestion
worker.start()
```